daemon-scheduler/pkg/regex/validate.go

```diff
@@ -0,0 +1,34 @@
+// Copyright 2016 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License"). You may
+// not use this file except in compliance with the License. A copy of the
+// License is located at
+//
+//     http://aws.amazon.com/apache2.0/
+//
+// or in the "license" file accompanying this file. This file is distributed
+// on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+// express or implied. See the License for the specific language governing
+// permissions and limitations under the License.
+
+package regex
```

Comment:
I'm still not convinced. We use these regexes for more than just validaing. For example, in CSS, we use them to define router paths. It'll be weird to use validate.ClusterARNRegex in the router because we're not validating anything. I'd be happy to change this to something more generic than 'validate'. Not sure I can think of anything though.